.github/workflows/docker.yml

```
@@ -1,92 +1,135 @@
-name: Publish Docker images
+name: Manual Docker Build
 
 on:
+  # 提交时自动触发（但排除同步机器人的提交）
   push:
-    tags:
-      - '*'
+    分支:
```

actionlint (1.7.7)

unexpected key "分支" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)